web/src/compose_closed_ui.ts

```diff
@@ -32,30 +40,24 @@ export function get_recipient_label(message?: ComposeClosedMessage): string {
             if (narrow_state.narrowed_to_topic()) {
                 const stream = narrow_state.stream_sub();
                 const stream_id = stream?.stream_id;
```

Comment:
This can be an assert, given `narrow_state.narrowed_to_topic`, and then you don't need the `if` below, I think -- if we're narrowed to a topic, we know there's in fact a stream/topic there to access.